streampark-console/streampark-console-newui/src/views/system/variable/components/VariableDrawer.vue

```diff
@@ -114,6 +115,17 @@
         componentProps: { rows: 4 },
         rules: [{ max: 100, message: t('system.variable.form.descriptionMessage') }],
       },
+      {
+        field: 'sensitive',
+        label: 'Sensitive',
```

Comment:
how about replace Sensitive to Datamask?

Reply:
Is it just that the front-end Sensitive is changed to Datamask, and the database fields and the fields of the back-end classes need to be changed?

Reply:
`Datamask` is a replacement for sensitive values, `Sensitive` refers to whether it is sensitive, so change `Sensitive` to `Datamask`, is it correct?

Reply:
> `Datamask` is a replacement for sensitive values, `Sensitive` refers to whether it is sensitive, so change `Sensitive` to `Datamask`, is it correct?

`Desensitization` is better, I recommend the front-end and back-end all replace to `Desensitization `.